java/src/jmri/Timebase.java

```diff
@@ -17,25 +20,25 @@
  * receive change notifications if the change is a minute or more, because it
  * changes continuously; query to {@code time} property when needed to get the
  * current value.
- * 
+ *
  * @author Bob Jacobsen Copyright (C) 2004, 2007, 2008
  */
 public interface Timebase extends NamedBean {
```

Comment:
Would it make sense to leave deprecation comments on this class (or its methods)?  Something like
```
@Deprecated(since="5.15.1", forRemoval=false) // used in scripts
 @deprecated Use {@code jmri.time.TimeProvider} instead
```

Reply:
Yes, absolutely. The problem is that deprecating it now causes lots of static analysis failures. So I think this needs to be done in steps. First step is to ensure TimeProvider actually works. Currently CI fails because I haven't implemented it fully. So I have the deprecation but commented out at the moment.